ACPs/84-table-preamble/README.md

```diff
@@ -0,0 +1,62 @@
+```text
+ACP: 84
+Title: Table Preamble for ACPs
+Author(s): Gauthier Leonard <https://github.com/Nuttymoon>
+Discussions-To: https://github.com/avalanche-foundation/ACPs/discussions/85
```

Comment:
This discussion link will be set when the ACP is merged by the admin.

Reply:
Example: https://github.com/avalanche-foundation/ACPs/pull/79

Reply:
Yeah, I didn't really know what to put until then. Should I leave this blank?

Reply:
Done